streams/src/main/java/org/apache/kafka/streams/kstream/KStreamBuilder.java

```diff
@@ -89,6 +99,7 @@
         return stream(offsetReset, null, null, topics);
```

Comment:
update to `return stream(offsetReset, null, null, null, topics);` to avoid too many indirections. To this for other overloads, too, please.